### ISSUE TITLE
opam init ERROR

### ISSUE THREAD

**Issue by ggxcarl** [ocaml/opam#2690]
```
Checking for available remotes: rsync and local, git.
  - you won't be able to use mercurial repositories unless you install the hg
    command on your system.
  - you won't be able to use darcs repositories unless you install the darcs
    command on your system.

=-=- Fetching repository information =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=  🐫 
[ERROR] Too many downloaded files.
[ERROR] Initialisation failed
# opam-version    1.2.2
# os              darwin
Too many downloaded files.
# opam-version    1.2.2
# os              darwin
```

**Comment by AltGr** [comment]
Please provide more details (at least `--debug`, `--verbose`) and reopen

**Comment by ggxcarl** [comment]
Guoxin-Macbook:~ Guoxin$ opam init --debug --verbose
00:00.005  CLIENT                  INIT default(0 http https://opam.ocaml.org)
Checking for available remotes: 00:00.005  SYSTEM                  mkdir /Users/Guoxin/.opam/log
rsync and local, git.
  - you won't be able to use mercurial repositories unless you install the hg
    command on your system.
  - you won't be able to use darcs repositories unless you install the darcs
    command on your system.

00:00.073  SYSTEM                  [log-3840-1872ec] (in 0.002s) ocamlc -version
00:00.074  STATE                   create-system-compiler-description 4.03.0
00:00.086  SYSTEM                  [log-3840-6d07ae] (in 0.001s) ocamlc -where
00:00.087  SYSTEM                  mkdir /Users/Guoxin/.opam/compilers
00:00.087  SYSTEM                  mkdir /Users/Guoxin/.opam/compilers/system
00:00.087  SYSTEM                  mkdir /Users/Guoxin/.opam/compilers/system/system
00:00.087  FILE(comp)              Wrote ~/.opam/compilers/system/system/system.comp in 0.000s
00:00.087  FILE(config)            Wrote ~/.opam/config in 0.000s
00:00.087  FILE(aliases)           Wrote ~/.opam/aliases in 0.000s
00:00.087  SYSTEM                  mkdir /Users/Guoxin/.opam/repo
00:00.087  FILE(repo-index)        Wrote ~/.opam/repo/package-index in 0.000s
00:00.088  FILE(repo-index)        Wrote ~/.opam/repo/compiler-index in 0.000s
00:00.088  SYSTEM                  mkdir /Users/Guoxin/.opam/repo/default
00:00.088  FILE(repo-config)       Wrote ~/.opam/repo/default/config in 0.000s
00:00.088  REPOSITORY              init default(0 http https://opam.ocaml.org)
00:00.088  SYSTEM                  rmdir /Users/Guoxin/.opam/repo/default
00:00.092  SYSTEM                  mkdir /Users/Guoxin/.opam/repo/default
00:00.092  FILE(repo-config)       Wrote ~/.opam/repo/default/config in 0.000s
00:00.092  SYSTEM                  mkdir /Users/Guoxin/.opam/repo/default/packages
00:00.092  SYSTEM                  mkdir /Users/Guoxin/.opam/repo/default/archives
00:00.092  SYSTEM                  mkdir /Users/Guoxin/.opam/repo/default/compilers
00:00.092  STATE                   install_global_config switch=system
00:00.093  SYSTEM                  mkdir /Users/Guoxin/.opam/system
00:00.093  SYSTEM                  mkdir /Users/Guoxin/.opam/system/config
00:00.094  FILE(.config)           Wrote ~/.opam/system/config/global-config.config in 0.000s
00:00.094  SYSTEM                  mkdir /Users/Guoxin/.opam/packages
00:00.094  CLIENT                  updating repository state
00:00.094  STATE                   LOAD-STATE(init-1)
00:00.094  FILE(config)            Wrote ~/.opam/config in 0.000s
00:00.094  CUDF                    Checking version of criteria accepted by the external solver
00:00.099  SYSTEM                  [log-3840-e7966f] (in 0.001s) aspcud -v
00:00.099  CUDF                    Solver is aspcud > 1.9: using latest version criteria
00:00.100  STATE                   ROOT      : /Users/Guoxin/.opam
00:00.100  STATE                   SWITCH    : system
00:00.100  STATE                   COMPILER  : system
00:00.100  STATE                   COMPILERS : { system }
00:00.100  STATE                   REPOS     : { default }
00:00.100  STATE                   PACKAGES  : {}
00:00.100  STATE                   INSTALLED : {}
00:00.100  STATE                   ROOTS     : {}
00:00.100  STATE                   REINSTALL : {}
00:00.100  STATE                   State init-1 loaded in 0.002s

=-=- Fetching repository information =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
00:00.100  REPOSITORY              update default(0 http https://opam.ocaml.org)
00:00.100  CURL                    pull-repo
00:00.100  SYSTEM                  error: File /Users/Guoxin/.opam/repo/default/urls.txt does not exist
00:00.100  CURL                    Rebuilding urls.txt at /Users/Guoxin/.opam/repo/default
00:00.100  FILE(file_attributes)   Wrote ~/.opam/repo/default/urls.txt in 0.000s
00:00.105  SYSTEM                  mkdir /var/folders/lz/qwvj3zp92z141f0rf1qtndh40000gn/T/opam-3840-d2d111
Processing: [default: http]
00:00.110  SYSTEM                  error: Too many downloaded files.
00:00.110  SYSTEM                  rmdir /var/folders/lz/qwvj3zp92z141f0rf1qtndh40000gn/T/opam-3840-d2d111
[ERROR] Too many downloaded files.
[ERROR] Initialisation failed
# opam-version    1.2.2
# os              darwin
Too many downloaded files.
'opam init --debug --verbose' failed.
# opam-version    1.2.2
# os              darwin

**Comment by AltGr** [comment]
Thanks. Could you check that `curl https://opam.ocaml.org/urls.txt` works ?
